README.md

```diff
@@ -79,306 +79,273 @@ int main()
+> you will need to specify C++ version via `CMAKE_CXX_STANDARD`
+> when manually configuring the project.
 
-<details>
-<summary> Disable tests build </summary>
+### Project specific configure arguments
 
-To build this project with tests disabled (and their dependencies),
-simply use `BEMAN_EXEMPLAR_BUILD_TESTING=OFF` as documented in upstream [CMake documentation](https://cmake.org/cmake/help/latest/module/CTest.html):
+When configuring the project manually,
+you can pass an array of project specific CMake configs to customize your build.
 
-```shell
-cmake -B build -S . -DBEMAN_EXEMPLAR_BUILD_TESTING=OFF
+Project specific options are prefixed with `BEMAN_EXEMPLAR`.
+You can see the list of available options with:
+
+```bash
+cmake -LH | grep "BEMAN_EXEMPLAR" -C 2
 ```
 
-</details>
+<details>
 
-## Integrate beman.exemplar into your project
+<summary> Details of CMake arguments. </summary>
 
-<details>
-<summary> Use beman.exemplar directly from C++ </summary>
-<!-- TODO Darius: rewrite section!-->
+#### `BEMAN_EXEMPLAR_BUILD_TESTS`
 
-If you want to use `beman.exemplar` from your project,
-you can include `beman/exemplar/*.hpp`  files from your C++ source files
+Enable building tests and test infrastructure. Default: ON.
+Values: { ON, OFF }.
 
-```cpp
-#include <beman/exemplar/identity.hpp>
+You can configure the project to have this option turned off via:
+
+```bash
+cmake -B build -S . -DCMAKE_CXX_STANDARD=20 -DBEMAN_EXEMPLAR_BUILD_TESTS=OFF
 ```
 
-and directly link with `libbeman.exemplar.a`
+> [!TIP]
+> Because this project requires Google Tests as part of its development
+> dependency,
+> disable building tests avoids the project from pulling Google Tests from
+> GitHub.
 
-```shell
-# Assume /opt/beman.exemplar staging directory.
-$ c++ -o identity_usage examples/identity_usage.cpp \
-    -I /opt/beman.exemplar/include/ \
-    -L/opt/beman.exemplar/lib/ -lbeman.exemplar
-```
+#### `BEMAN_EXEMPLAR_BUILD_EXAMPLES`
+
+Enable building examples. Default: ON. Values: { ON, OFF }.
 
 </details>
 
-<details>
-<summary> Use beman.exemplar directly from CMake </summary>
+## Integrate beman.exemplar into your project
 
-<!-- TODO Darius: rewrite section! Add examples. -->
+To use `beman.exemplar` in your C++ project,
+include an appropriate `beman.exemplar` header from your source code.
 
-For CMake based projects, you will need to use the `beman.exemplar` CMake module to define the `beman::exemplar` CMake target:
+```c++
+#include <beman/exemplar/identity.hpp>
+```
+
+> [!NOTE]
+>
+> `beman.exemplar` headers are to be included with the `beman/exemplar/` directories prefixed.
+> It is not supported to alter include search paths to spell the include target another way. For instance,
+> `#include <identity.hpp>` is not a supported interface.
+
+How you will link your project against `beman.exemplar` will depend on your build system.
+CMake instructions are provided in following sections.
+
+### Linking your project to beman.exemplar with CMake
```

Comment:
"Linking" has a specific meaning in C++. I think we should use "incorporate" instead.